plugin.video.montreal.greek-tv/resources/lib/default.py

```diff
@@ -33,7 +34,9 @@ def playlinks(url):
         liz = xbmcgui.ListItem(offscreen=True)
         liz.setProperty('isFolder', 'false')
         if url == RADIO_URL :
+            if xbmc.Player().isPlayingVideo() == True : xbmc.executebuiltin("Action(Stop)")
```

Comment:
`is True`

Reply:
it was by inattention that I wrote it this way. I replaced it.
Thank you